addons/survey/views/survey_templates_print.xml

```diff
@@ -34,12 +34,62 @@
                                             <span t-if="question.constr_mandatory" class="text-danger">*</span>
                                             <span t-if="scoring_display_correction" class="badge badge-pill" t-att-data-score-question="question.id"></span>
                                         </h2>
-                                        <t t-if="question.description"><div class="text-muted oe_no_empty" t-field='question.description'/></t>
-                                        <t t-if="question.question_type == 'text_box'"><t t-call="survey.question_text_box"/></t>
-                                        <t t-if="question.question_type == 'char_box'"><t t-call="survey.question_char_box"/></t>
-                                        <t t-if="question.question_type == 'numerical_box'"><t t-call="survey.question_numerical_box"/></t>
-                                        <t t-if="question.question_type == 'date'"><t t-call="survey.question_date"/></t>
-                                        <t t-if="question.question_type == 'datetime'"><t t-call="survey.question_datetime"/></t>
+                                        <div class="text-muted oe_no_empty mt-1" t-if="not is_html_empty(question.description)" t-field='question.description'/>
+                                        <t t-if="question.question_type in ['numerical_box', 'date', 'datetime', 'text_box', 'char_box']">
+                                            <t t-if="answer_lines">
+                                                <t t-set="answer_line" t-value="answer_lines[0]"/>
+                                                <t t-if="answer_line.skipped">
+                                                    <!-- question was skipped, display an orange block with the text "Skipped" in place of the answer -->
+                                                    <div class="row no-gutters">
+                                                        <t t-set="answer_bg_class" t-value="'o_survey_question_skipped'"/>
+                                                        <div t-attf-class="col-12 col-md-6 col-lg-4 rounded pl-4 {{answer_bg_class}}">
+                                                            <input type="text"
+                                                                t-attf-class="form-control font-italic o_survey_question_{{question.question_type}} bg-transparent rounded-0 p-0"
+                                                                name="skipped_answers" value="Skipped"/>
+                                                        </div>
+                                                    </div>
+                                                </t>
+                                                <t t-elif="not answer_line.skipped and answer_lines.survey_id.scoring_type != 'no_scoring' and question.question_type in ['numerical_box', 'date', 'datetime']">
```

Comment:
Shouldn't we additionally test that there is a correct answer defined with:

`question['answer_%s' % question.question_type]` as below?

Currently it will show up in red even if there is no correct answer, see:

![image](https://user-images.githubusercontent.com/41282536/162908158-4fabd9d1-8781-494d-ac81-d6470530d624.png)

Here my first question does not have a "correct answer" set up.